src/PscIde/Command.purs

```diff
@@ -277,9 +277,16 @@ instance decodePursuitCompletion :: DecodeJson PursuitCompletion where
       })
 
 instance decodeImportList :: DecodeJson ImportList where
-  decodeJson json = do
-    imports <- decodeJson json
-    pure (ImportList imports)
+  decodeJson json = decodeObject <|> decodeArray
+    where
+      decodeObject = do
+        o <- decodeJson json
+        moduleName <- o .? "moduleName"
+        imports <- o .? "imports"
+        pure (ImportList { moduleName: head moduleName, imports })
```

Comment:
Turns out I'm an idiot... https://github.com/purescript/purescript/blob/6dc0e2ec2fdf30ab815af0151d6f9c6e3fbc1155/src/Language/PureScript/Ide/Types.hs#L232 uses the ToJSON instance for `ModuleName` which will probably return something like `["Data", "Array"]`. I'll send a patch to turn that into Text before encoding right away, but we might need to support this bug until then... sorry.